### PR TITLE
RDoc-2134-RDoc-2196-RDoc-2207 Fixed typos and updated formatting

### PR DESCRIPTION
**Pull request by serezhub**
These are 3 typo/formatting-related issues pushed into one PR to save time.

### CODE REVIEW

File: Documentation/5.2/Raven.Documentation.Pages/server/security/authentication/certificate-management.markdown [hunk 1]
```diff
@@ -24,11 +27,9 @@ In this page:
   * [Private Keys](../../../server/security/authentication/certificate-management#private-keys)  
   * [Client Certificate Chain of Trust](../../../server/security/authentication/certificate-management#client-certificate-chain-of-trust)  
 
+{NOTE/ }
```
Comment (reebhub) [inline Documentation/5.2/Raven.Documentation.Pages/server/security/authentication/certificate-management.markdown:30]: this `{NOTE/ }` is shown as a command in the final page, and this is not the only such occurrence in this page.
there are **at least 5 other places** where NOTE or PANEL commands are displayed as such in the final page.

File: Documentation/5.2/Raven.Documentation.Pages/server/security/authentication/certificate-management.markdown [hunk 2]
```diff
@@ -75,11 +74,11 @@ does not really exist within RavenDB in this manner. Instead, cluster admins or
 
 
 * [Cluster Admin](../../../server/security/authorization/security-clearance-and-permissions#cluster-admin)  
-  Full administrative access to cluster and databases within.  
+  Full administrative access to clusters and databases within.  
 * [Operator](../../../server/security/authorization/security-clearance-and-permissions#operator)  
   Admin access to databases, but not to modify the cluster.  
 * [User](../../../server/security/authorization/security-clearance-and-permissions#user)  
-  Lowest levels of privelages. 
+  Lowest levels of priveleges.
```
Comment (reebhub) [inline Documentation/5.2/Raven.Documentation.Pages/server/security/authentication/certificate-management.markdown:81]: you replaced one mistake with another.
--> privileges

Comment (reebhub) [reply Documentation/5.2/Raven.Documentation.Pages/server/security/authentication/certificate-management.markdown:81]: this and the note/panel issue means you probably didn't run any spell checking. please do.

**Review by reebhub — CHANGES_REQUESTED** [review]

File: Documentation/5.2/Raven.Documentation.Pages/server/security/authentication/certificate-management.markdown (hunk 1 above)
Comment (serezhub) [reply Documentation/5.2/Raven.Documentation.Pages/server/security/authentication/certificate-management.markdown:30]: Done. 
I wonder when I put a space after the / {PANEL/ }.

File: Documentation/5.2/Raven.Documentation.Pages/server/security/authentication/certificate-management.markdown (hunk 2 above)
Comment (serezhub) [reply Documentation/5.2/Raven.Documentation.Pages/server/security/authentication/certificate-management.markdown:81]: I just did it again.  'Privileges' was the only word that slipped by.

**Review by reebhub — APPROVED** [review]

**Pull request merged**